src/NLog/Targets/FileTarget.cs

```diff
@@ -971,8 +995,8 @@ private void ResetFileAppenders(string reason)
             }
         }
 
-        private readonly ReusableStreamCreator _reusableFileWriteStream = new ReusableStreamCreator(4096);
-        private readonly ReusableStreamCreator _reusableAsyncFileWriteStream = new ReusableStreamCreator(4096);
+        private ReusableStreamCreator _reusableFileWriteStream;
```

Comment:
Like the default-capacity of 4K.

Reply:
It's still 4K by default. We set it in contructor